### ISSUE TITLE
[Tutorial] 86Box

### ISSUE THREAD

**Issue by pulkitkrishna00** [InstallerLegacy/setup.virtualhub.eu.org#13]
- Homepage: <https://86box.net>

Information:
- Appimage
- Flatpak

Status:

- [x] Tutorial
- [ ] Video

Depends on: #4

**Comment by pulkitkrishna00** [comment]
Added tutorial: https://setup.virtualhub.eu.org/86box/

**Comment by pulkitkrishna00** [comment]
Closing this, as it has been decided that a separate issue will be created for video.
